README.md

```diff
@@ -82,42 +64,35 @@ For example, you can create business models with different pricing tiers, e.g.:
 
 ### That's it
 
-That's it, you're now ready to earn recurring revenue from your customers \o/
+That's it, you're now ready to earn recurring revenue from your customers 🥳
 
-## Develop locally
+## Going live
 
-If you've deployed the project with Vercel, it will have created a repository for you which you can clone to your local machine. If you haven't deployed with Vercel, you can use [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) or [npx](https://github.com/zkat/npx#readme) to bootstrap the example:
+To run the project in live mode and process real payments with Stripe, you can add the [Stripe Go Live](https://vercel.com/integrations/stripe-live) integration to your project. This will set up your live API keys and webhook secret as environment variables for the production environment.
 
-```bash
-npx create-next-app --example https://github.com/thorwebdev/nextjs-subscription-payments my-saas-app
-# or
-yarn create next-app --example https://github.com/thorwebdev/nextjs-subscription-payments my-saas-app
-```
+Afterward you will need to rebuild your production deployment for the changes to take effect. Within your project Dashboard, navigate to the "Deployments" tab, select the most recent deployment, click the overflow menu button (next to the "Visit" button) and select "Redeploy".
```

Comment:
Could we make the live integration kick off a deployment?

Reply:
Yes, but this one we wouldn't want to do automatically, right, but rather on a button click? I'm just thinking that some folks might want to make some other modifications before triggering a prod deployment. Wdyt?

Reply:
Yeah thinking about this more it's probably overkill. It just needs to be clear that a redeploy is required.

If they've made it to this part, then a redeploy is the least of their concerns!